.github/workflows/tests.yaml

```
@@ -0,0 +1,45 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  python:
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Quotes are needed otherwise it tries to test Python 3.1 (https://dev.to/hugovk/the-python-3-1-problem-85g)
```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10"]
```